.github/workflows/publish-docker.yml

```diff
@@ -51,7 +55,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: ${{ env.REGISTRY }}/${{ github.repository }}/${{ matrix.binary }}
+          images: ${{ env.REGISTRY }}/${{ github.repository }}/${{ matrix.binary if matrix.features != 'integration-tests' else 'tangle-integration-tests' }}
```

Comment:
And Here you can just use `${{ matrix.image }}`

Reply:
I had the same thoughts.. Updated